layouts/partials/compliance/controls.html

```diff
@@ -0,0 +1,29 @@
+<section class="mt-8 py-4 px-4 md:px-0">
+    <div class="container mx-auto max-w-4xl">
+        <h3>What is {{ .Params.service }}?</h3>
+        <p>
+            {{ .Params.whatis }}
+        </p>
+    </div>
+</section>
+<section class="mt-4 py-4 px-4 md:px-0">
+    <div class="container mx-auto max-w-4xl">
+        <h3>What controls can I put in place to evaluate {{ .Params.service }} resources?</h3>
```

Comment:
Ok this the way this question is written, the bullet points don't grammatically work to answer the questions. So we either have to change the question or change how the bullets are written. I suspect the latter is harder.

Reply:
Ok, if we change the bullets, we can replace "should not be" with "are not" and "should be" with "are"

Reply:
I actually shared the same opinion with regards to the grammatical correctness of this when I first created this section of the page and read it through. Though I eventually became convinced it sounds fine. These are the actual controls defined by aws and a control is basically a check to put in place to evaluate the resource's state of compliance. So for example when the question is "What controls can I put in place to evaluate X resources?",  and as an example a control is "Security groups should not allow ingress from 0.0.0.0/0 or ::/0 to port 22". This reads right to me as in you would have a check to ensure that security groups don't allow ingress from everywhere.

Yeah it would definitely by more difficult to change the bullets because it cannot be automated and needs to be human evaluated for correctness since substituting "should not be" for "are not" may also mean we need to change other words in the sentence around it for it to be correct. Currently, bringing in these bullet points is automated by running a script that scrapes them in and adds them to the json. But we can correct them after they are brought in if you want.